Controller.php

```diff
@@ -115,6 +115,28 @@ public function index($errorMessage = false)
         ]);
     }
 
+    public function forwardToAuth()
+    {
+        Piwik::checkUserHasSuperUserAccess();
+
+        Nonce::checkNonce('gaimport.auth', Common::getRequestVar('auth_nonce'));
+
+        $session = $this->getSession();
+
+        $session->gaauthtime = time();
+        $session->setExpirationSeconds(60 * 15, 'gaauthtime');
```

Comment:
@sgiehl can you make the same change as in the other PR?

Reply:
@diosmosis done.